templates/AGENTS-security.md

```
@@ -0,0 +1,221 @@
+- HTTP (not HTTPS) for any external API call or redirect
+- Regular expressions susceptible to ReDoS (catastrophic backtracking)
+- File operations without path canonicalization (normalize + resolve before access)
+
+## Severity Classification
+
+| Severity | Criteria |
+|----------|----------|
+| CRITICAL | Remote code execution, auth bypass, secret exposure, SQL injection with data exfiltration |
+| HIGH | Privilege escalation, IDOR with sensitive data, broken crypto, missing auth on sensitive endpoint |
+| MEDIUM | Missing rate limiting, information disclosure, weak logging, missing security headers |
+| LOW | Missing input validation on non-sensitive field, verbose error messages, minor config issues |
```

markdownlint-cli2 (0.23.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)